prisma/schema.prisma

```diff
@@ -7,50 +7,70 @@ datasource db {
 }
 
 generator client {
-  provider        = "prisma-client-js"
+  provider = "prisma-client-js"
 }
 
-model Account {
-  id                 String  @id @default(cuid())
-  userId             String
-  type               String
-  provider           String
-  providerAccountId  String
-  refresh_token      String?  @db.Text
-  access_token       String?  @db.Text
-  expires_at         Int?
-  token_type         String?
-  scope              String?
-  id_token           String?  @db.Text
-  session_state      String?
-
-  user User @relation(fields: [userId], references: [id], onDelete: Cascade)
+model Survey {
+  id        String   @id @default(cuid())
+  name      String
+  createdAt DateTime @default(now())
+  userId    String
+  questions Question[]
+  user      User     @relation(fields: [userId], references: [id])
+}
 
-  @@unique([provider, providerAccountId])
+model Question {
+  id        String   @id @default(cuid())
+  content   String
```

Comment:
What exactly is the `content` of a question? Please think about the mockup that we created. Each Question should have a title, optional description and some attributes for the slider (steps, min, max). Please add that to the model as well